File: src/main/java/by/architecture_map/belarus/entity/Address.java
package by.architecture_map.belarus.entity;

import jakarta.persistence.Entity;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Entity
@Data
@EqualsAndHashCode(callSuper = true)
public class Address extends BaseEntity {
    private String locality;
    private String district;
    private String region;
    private String street;
    private String houseNumber;
}

File: src/main/kotlin/by/architecture_map/belarus/entity/Address.kt
package by.architecture_map.belarus.entity

import jakarta.persistence.Entity

@Entity
data class Address(

    var locality: String? = null,
    var district: String? = null,
    var region: String? = null,
    var street: String? = null,
    var houseNumber: String? = null

) : BaseEntity()
